### PR TITLE
Release 2.22.0

### PR DESCRIPTION
**Pull request by ydah**
______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [-] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Master (Unreleased)
 
-## 2.21.0 (2023-XX-XX)
+## 2.21.0 (2023-05-03)
 
 - Extracted from `rubocop-rspec` into a separate repository for easier use with Minitest/Cucumber. ([@ydah])
```
Comment (pirj) [inline CHANGELOG.md:8]: Below is pretty old, we should refer to a more recent ref like 6b1ea4c3.

**Review by pirj — APPROVED** [review]
Thank you! 🎉 
Looks good unless I'm missing something.

File: CHANGELOG.md (hunk 1 above)
Comment (ydah) [reply CHANGELOG.md:8]: Being old means we need to cherry-pick some commits? Or am I missing something?

Comment (pirj) [reply CHANGELOG.md:8]: The link a few lines below that suggest checking earlier changelog in rubocop-r spec points to a version when we extracted rubocop-capybara. We should point to a more recent rubocop-rspec changelog revision that would include changes for 2.20.0, as it has some changes to factory_bot cops.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Master (Unreleased)
 
-## 2.21.0 (2023-XX-XX)
+## 2.21.0 (2023-05-03)
```
Comment (bquorning) [inline CHANGELOG.md:5]: Perhaps we should do a v2.22.0 instead. Then we can do a v2.21.0 release of rubocop-rspec [without the removal of factory_bot cops](https://github.com/rubocop/rubocop-rspec/blob/70a97b1895ce4b9bcd6ff336d5d343ddc6175fe6/CHANGELOG.md), followed by a v2.22.0 release with just the removal of factory_bot?

File: CHANGELOG.md (hunk 1 above)
Comment (ydah) [reply CHANGELOG.md:8]: Ah! I see😅 I had completely missed that. Thank you ❤️

File: CHANGELOG.md (hunk 2 above)
Comment (ydah) [reply CHANGELOG.md]: Indeed it is! Change to v2.22.0 👍🏻

**Review by bquorning — APPROVED** [review]
This PR goes hand in hand with https://github.com/rubocop/rubocop-rspec/pull/1583, correct?

**Comment by ydah** [comment]
Yes. After this release, we can proceed with the release by freezing the release version of rubocop-factory_bot at https://github.com/rubocop/rubocop-rspec/pull/1583 👍

**Review by pirj — APPROVED** [review]
Looks good! :shipit:

**Comment by ydah** [comment]
@pirj If possible, would you be willing to publish to https://rubygems.org/gems/rubocop-factory_bot ?

I think you know @bquorning , @Darhazer and my email address during the extraction of rubocop-capybara, so please invite to the gem owner 🙏🏻

**Comment by pirj** [comment]
✅ Pushed [2.22.0 to RubyGems](https://rubygems.org/gems/rubocop-factory_bot/versions/2.22.0).
✅ Added all of us as [gem owners on RubyGems](https://rubygems.org/gems/rubocop-factory_bot). Please tell me if you're missing an invitation from RubyGems.

**Comment by ydah** [comment]
@pirj Thank you ❤️

**Pull request merged**